### PR TITLE
Fix data fixture namespace

### PR DESCRIPTION
**Pull request by hluchas**
| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | staging
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 

#### Description:
There's a bug in namespace case which prevents test to be run on systems with case sensitive filesystem such as OSX.

#### Steps to test this PR:
1. Ensure that tests are passing on OSX

### CODE REVIEW

**Comment by hluchas** [comment]
I don't understand why unchanged file has problem with 
```
   1) mautic/app/bundles/FormBundle/Helper/FormUploader.php (yoda_style)
      ---------- begin diff ----------
--- Original
+++ New
@@ @@
     {
-        if (exif_imagetype($filename) != IMAGETYPE_JPEG) {
+        if (IMAGETYPE_JPEG != exif_imagetype($filename)) {
             return;
         }
```

The second thing is that this file does not contain such condition in this branch.
https://github.com/hluchas/mautic/blob/fix-data-fixture-namespace/app/bundles/FormBundle/Helper/FormUploader.php

EDIT: I changed target branch to `staging` because I used it to create new branch even if this is bugfix. This will probably need Travis job run again

**Comment by mabumusa1** [comment]
@hluchas do you need help to get this PR ready?

**Comment by hluchas** [comment]
> @hluchas do you need help to get this PR ready?

I need help from somebody who could explain what's wrong with CI and fix this.

**Comment by mabumusa1** [comment]
@hluchas you need to fix the code style, please run the command 

`php bin/php-cs-fixer fix -v` this will fix the styles if that did not work for some reason, just change this line

` if (exif_imagetype($filename) != IMAGETYPE_JPEG) {`
to
` if (IMAGETYPE_JPEG != exif_imagetype($filename)) {`

**Comment by hluchas** [comment]
> @hluchas you need to fix the code style, please run the command
> 
> `php bin/php-cs-fixer fix -v` this will fix the styles if that did not work for some reason, just change this line
> 
> ` if (exif_imagetype($filename) != IMAGETYPE_JPEG) {`
> to
> ` if (IMAGETYPE_JPEG != exif_imagetype($filename)) {`

As I mentioned here https://github.com/mautic/mautic/pull/9297#issuecomment-708324891
`this file does not contain such condition in this branch.`

I other words. There's nothing to change in this branch.

**Comment by mabumusa1** [comment]
@hluchas 
You can rebase your code and push again, I think this issue will be solved.

https://contribute.mautic.org/contributing-to-mautic/developer/code/pull-requests#rebase-your-pull-request

**Comment by hluchas** [comment]
> @hluchas
> You can rebase your code and push again, I think this issue will be solved.
> 
> https://contribute.mautic.org/contributing-to-mautic/developer/code/pull-requests#rebase-your-pull-request

done. There's no change

```
13:45 $ git status
On branch staging
Your branch is up to date with 'origin/staging'.

nothing to commit, working tree clean
✔ ~/dev/mautic-community [staging|✔]
13:46 $ git fetch origin
✔ ~/dev/mautic-community [staging|✔]
13:46 $ git merge origin/staging
Already up to date.
✔ ~/dev/mautic-community [staging|✔]
13:46 $ git checkout fix-data-fixture-namespace
Switched to branch 'fix-data-fixture-namespace'
Your branch is up to date with 'hluchas/fix-data-fixture-namespace'.
✔ ~/dev/mautic-community [fix-data-fixture-namespace|✔]
13:46 $ git rebase staging
Current branch fix-data-fixture-namespace is up to date.
✔ ~/dev/mautic-community [fix-data-fixture-namespace|✔]
13:47 $ git push
Everything up-to-date
✔ ~/dev/mautic-community [fix-data-fixture-namespace|✔]
13:47 $
```

**Comment by escopecz** [comment]
Closing and opening to trigger Travis rebuild again.

**Comment by escopecz** [comment]
Testing if changing branch to staging will fix CodeCov

**Review by escopecz — APPROVED** [review]
This is needed to fix tests on local (mac)

File: app/release_metadata.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "version": "3.1.2-rc",
-  "stability": "rc",
+  "version": "3.1.2",
+  "stability": "stable",
```
Comment (escopecz) [inline app/release_metadata.json]: Wait, why this change?

**Review by escopecz — DISMISSED** [review]

File: app/release_metadata.json (hunk 1 above)
Comment (fedys) [reply app/release_metadata.json]: @hluchas please remove this change as this is blocking the PR.

Comment (npracht) [reply app/release_metadata.json]: Hm, it comes from one of my commits, I don't know why. It should not be brought by a PR. I think we can just revert this commit. I think it came upper when we did some maintenance on branches and the conflicts has probably solved wrongly.

Can we just remove this commit ?

Comment (npracht) [reply app/release_metadata.json]: Okay, earlier this morning i was preparing 3.2.0-rc (mistake, the date have been moved to later). So i changed this file not in the right timing. I just changed back the file on staging.

Comment (npracht) [reply app/release_metadata.json]: Just added one commit here to be fully equal to staging, i think now @escopecz we could consider this a resolved ?

**Review by npracht — DISMISSED** [review]

**Review by fedys — DISMISSED** [review]

File: app/release_metadata.json (hunk 1 above)
Comment (hluchas) [reply app/release_metadata.json]: reverted

**Review by npracht — DISMISSED** [review]
👍

**Comment by hluchas** [comment]
Commits 
```
2785bde530fd048a40b31fd725a40a368b9e56ec
310818187744b6336704f9b406453ab633022649
```
are reverted

File: app/bundles/LeadBundle/Tests/Controller/LeadControllerTest.php [hunk 2]
```diff
@@ -3,7 +3,8 @@
 namespace Mautic\LeadBundle\Tests\Controller;
 
 use Doctrine\DBAL\Connection;
-use Mautic\CampaignBundle\Tests\DataFixtures\Orm\CampaignData;
+use Illuminate\Support\Collection;
```
Comment (escopecz) [inline app/bundles/LeadBundle/Tests/Controller/LeadControllerTest.php]: Why this line was added?
```suggestion
```

**Review by escopecz — DISMISSED** [review]

File: app/bundles/LeadBundle/Tests/Controller/LeadControllerTest.php (hunk 2 above)
Comment (hluchas) [reply app/bundles/LeadBundle/Tests/Controller/LeadControllerTest.php]: removed

**Review by escopecz — APPROVED** [review]
Thanks Lukas!

**Review by fedys — APPROVED** [review]
👍
